### PR TITLE
Working on Landing Page

### PR DESCRIPTION
**Pull request by namanhkapur**

### CODE REVIEW

**Comment by gvacaliuc** [comment]
Would you mind putting screenshots of before after for frontend PRs so we can quickly see what the expected change is?

File: client/src/App.css [hunk 1]
```diff
@@ -1,22 +1,40 @@
...
+    justify-content: center;
+  }
+  
+  h1 {
+    padding-top: 120px;
+    font-size: 75px;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    font-style: italic;
+  }
+
```
Comment (rgreen312) [inline client/src/App.css]: nit: new line at the end of files

File: client/src/App.css [hunk 2]
```diff
@@ -1,22 +1,40 @@
...
-  font-size: calc(10px + 2vmin);
-  color: white;
-}
-
-.App-link {
-  color: #09d3ac;
-}
+.top-nav-bar {
+    background-color: black;
+    width: 100%;
```
Comment (rgreen312) [inline client/src/App.css]: nit: let's try to alphabetize css properties

File: client/src/App.tsx [hunk 3]
```diff
@@ -1,26 +1,20 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <div className="top-nav-bar">
```
Comment (rgreen312) [inline client/src/App.tsx]: We should probably make this its own component, but if that's coming later I'm fine leaving it this way for now

**Pull request merged**